plugins/janus_sip.c

```diff
@@ -2464,6 +2470,7 @@ void janus_sip_sofia_callback(nua_event_t event, int status, char const *phrase,
 				JANUS_LOG(LOG_VERB, "Detected video codec: %d (%s)\n", session->media.video_pt, session->media.video_pt_name);
 			}
 			session->media.ready = 1;	/* FIXME Maybe we need a better way to signal this */
+			session->raw_media = parser;
```

Comment:
This made me notice that we're missing a `sdp_parser_free(parser);` there: we do have it for errors and not for when things go fine instead. As a side note, you're not freeing that reference either, which is a leak.